src/plugins/intel_cpu/tests/unit/snippets_transformations/aarch64/lowered/gemm_blocking.cpp

```diff
@@ -0,0 +1,224 @@
+    handlers.register_pass<SpecificLoopIterType::FIRST_ITER, ov::intel_cpu::pass::GemmCPUBlocking::DummyPass>();
+    return handlers;
+}
+
+void create_gemm_loop_infos(const LinearIRPtr& linear_ir,
+                             const ExpressionPtr& gemm_expr,
+                             size_t m = 0, size_t m_blk = 0,
+                             size_t k = 0, size_t k_blk = 0,
+                             size_t n = 0, size_t n_blk = 0) {
+    const bool k_block = k != 0 && k_blk != 0;
+    const bool n_block = n != 0 && n_blk != 0;
+    const bool m_block = m != 0 && m_blk != 0;
+    if (k_block) {
+        const auto loop_info =
+            std::make_shared<ov::snippets::lowered::UnifiedLoopInfo>(k, k_blk,
+                std::vector<LoopPort>{LoopPort::create<PortType::Incremented>(gemm_expr->get_input_port(0), 0),
+                                      LoopPort::create<PortType::Incremented>(gemm_expr->get_input_port(1), 1)},
+                std::vector<LoopPort>{LoopPort::create<PortType::NotProcessed>(gemm_expr->get_output_port(0))},
+                get_k_loop_handlers(k, k_blk));
+        linear_ir->get_loop_manager()->add_loop_info(loop_info);
+    }
+    if (n_block) {
+        linear_ir->get_loop_manager()->add_loop_info(
+            std::make_shared<ov::snippets::lowered::UnifiedLoopInfo>(n, n_blk,
+                std::vector<LoopPort>{LoopPort::create<PortType::NotProcessed>(gemm_expr->get_input_port(0)),
+                                      LoopPort::create<PortType::Incremented>(gemm_expr->get_input_port(1))},
+                std::vector<LoopPort>{LoopPort::create<PortType::Incremented>(gemm_expr->get_output_port(0))},
+                BrgemmBlockingBase::get_default_blocking_loop_handlers(n, n_blk)));
+    }
+    if (m_block) {
+        std::vector<LoopPort> entries{LoopPort::create<PortType::Incremented>(gemm_expr->get_input_port(0), 1)};
+        for (size_t i = 1; i < gemm_expr->get_input_count(); ++i)
+            entries.push_back(LoopPort::create<PortType::NotProcessed>(gemm_expr->get_input_port(i)));
+        linear_ir->get_loop_manager()->add_loop_info(
+            std::make_shared<ov::snippets::lowered::UnifiedLoopInfo>(m, m_blk,
+                entries,
+                std::vector<LoopPort>{LoopPort::create<PortType::Incremented>(gemm_expr->get_output_port(0), 1)},
+                BrgemmBlockingBase::get_default_blocking_loop_handlers(m, m_blk)));
+    }
+}
+
+} // namespace
+
+class GemmBlockingTest : public LoweredPassTestsF {
+public:
+    GemmBlockingTest() : LoweredPassTestsF() {
+        comparator.enable(LIRComparator::LIRCmpValues::LOOP_INDICES);
+        comparator.enable(LIRComparator::LIRCmpValues::PORT_DESCRIPTORS);
+        comparator.enable(LIRComparator::LIRCmpValues::PORT_CONNECTORS);
+        comparator.enable(LIRComparator::LIRCmpValues::LOOP_MANAGER);
+    }
+
+protected:
+    size_t m_blk = 32;
+    size_t k_blk = 512;
```

Comment:
`k_blk` currently equals to full dim. Please double check it in tests

Reply:
Done